<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>835</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>251</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QPushButton" name="BtnSimpan">
      <property name="text">
       <string>SIMPAN</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="BtnUbah">
      <property name="text">
       <string>UBAH</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="BtnHapus">
      <property name="text">
       <string>HAPUS</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="idMerekLabel">
      <property name="text">
       <string>Id Merek</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="idMerekLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="merekLabel">
      <property name="text">
       <string>Merek</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="merekLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineCari">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>70</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblMerek">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>130</y>
     <width>261</width>
     <height>221</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Id Merek</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Merek</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>551</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="filterDataLabel">
      <property name="text">
       <string>Filter Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboFilter">
      <item>
       <property name="text">
        <string>Semua</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Merek</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btnCetak">
      <property name="text">
       <string>Cetak</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="txtMerek"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
